### PR TITLE
OCM-9274 | RHCS: Add new profile jobs

### PR DESCRIPTION
**Pull request by radtriste**
- rosa-hcp-network
- rosa-hcp-encryption

https://issues.redhat.com/browse/OCM-9274

Depends on https://github.com/terraform-redhat/terraform-provider-rhcs/pull/710

### CODE REVIEW

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-encryption-optional-presubmit pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-ingress-optional-presubmit pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-network-optional-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-encryption-optional-presubmit pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-ingress-optional-presubmit pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-network-optional-presubmit

**Comment by radtriste** [comment]
Waiting for https://github.com/terraform-redhat/terraform-provider-rhcs/pull/709 to be merged and solve the test issue

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-encryption-optional-presubmit	 pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-network-optional-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-hcp-encryption-optional-presubmit

**Comment by radtriste** [comment]
@yasun1 @xueli181114 can I get approval ?

File: ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml [hunk 1]
```diff
@@ -478,6 +510,36 @@ tests:
     test:
     - ref: rhcs-e2e-tests
     workflow: rhcs-aws-sts
+- always_run: false
+  as: rosa-hcp-network-optional-presubmit
```
Comment (yasun1) [inline ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml:514]: Could you help me understand the meaning of `optional` here? 

> optional Set to true to make the job not block merges.

If you do not want this job to be a merge block in Prow CI, you should set `optional: true`.

**Comment by yasun1** [comment]
lgtm

**Review by ciaranRoche — APPROVED** [review]
/lgtm

**Pull request merged**